test/utils/storage.ts

```diff
@@ -0,0 +1,50 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import fs from "fs";
+
+type StateVariable = {
+  name: string;
+  slot: string;
+  offset: number;
+  type: string;
+};
+
+export const getContractStorageLayout = async (
```

Comment:
https://github.com/aurora-is-near/hardhat-storage-layout/blob/main/src/storageLayout.ts#L31 proudly inspired this